selfdrive/car/tests/routes.py

```diff
@@ -139,7 +139,7 @@
   CarTestRoute("ffccc77938ddbc44|2021-01-04--16-55-41", TOYOTA.CAMRYH_TSS2),
   CarTestRoute("54034823d30962f5|2021-05-24--06-37-34", TOYOTA.CAMRYH),
   CarTestRoute("4e45c89c38e8ec4d|2021-05-02--02-49-28", TOYOTA.COROLLA),
-  CarTestRoute("5f5afb36036506e4|2019-05-14--02-09-54", TOYOTA.COROLLA_TSS2),
+  CarTestRoute("6444d42c9eba0145|2022-11-09--18-05-27", TOYOTA.COROLLA_TSS2),
```

Comment:
```suggestion
  CarTestRoute("5f5afb36036506e4|2019-05-14--02-09-54", TOYOTA.COROLLA_TSS2),
```

same platform, no need to add/replace test routes